automation/vars/hci.yaml

```diff
@@ -76,3 +76,69 @@ vas:
           - name: edpm-deployment-values-post-ceph
             src_file: values.yaml
         build_output: deployment-post-ceph.yaml
+
+# OSP related content for adoption
+osp:
```

Comment:
Maybe we should call this directly "adoption" because we don't care about any "generic OSP configs" outside of adoption context.